Detections/ASimWebSession/PossibleDGAContacts.yaml

```diff
@@ -0,0 +1,104 @@
+id: 9176b18f-a946-42c6-a2f6-0f6d17cd6a8a
+name: Possible contact with a domain generated by a DGA
```

Comment:
Name: Potential communication with a Domain Generation Algorithm (DGA) based hostname

Description:
Identifies communication with hosts that have a domain name that might have been generated by a Domain Generation Algorithm (DGA). DGAs are used by malware to generate rendezvous points that are difficult to predict in advance. This detection uses the top 1 million domain names to build a model of what normal domains look like nad uses the model to identify domains that may have been randomly generated by an algorithm. You can modify the triThreshold and dgaLengthThreshold query parameters to change Analytic Rule sensitivity. The higher the numbers, the less noisy the rule is. \<br><br>To use this Analytics Rule, [deploy the Advanced SIEM information Model (ASIM)](https://aka.ms/DeployASIM\).

alertDisplayNameFormat: Potential communication from {{SrcIpAddr} with a Domain Generation Algorithm (DGA) based host {{Name}}

alertDescriptionFormat:
{{SrcIpAddr}} communicated with host {{Name}} that have a domain name that might have been generated by a Domain Generation Algorithm (DGA), identified by the pattern {{DGADomain}}. DGAs are used by malware to generate rendezvous points that are difficult to predict in advance. This detection uses the top 1 million domain names to build a model of what normal domains look like nad uses the model to identify domains that may have been randomly generated by an algorithm

Custom Fields:
DGADomain - "DGA pattern"